gatsby-config.js

```diff
@@ -70,6 +70,16 @@ module.exports = {
         path: `${__dirname}/src/content`,
       },
     },
+    // This plugin is set to disallow to prevent crawling of the site during preview
+    // mode
+    {
+      resolve: 'gatsby-plugin-robots-txt',
+      options: {
+        host: 'https://www.docs-preview.newrelic.com',
+        sitemap: 'https://www.docs-preview.newrelic.com/sitemap.xml',
```

Comment:
```suggestion
        host: 'https://docs-preview.newrelic.com',
        sitemap: 'https://docs-preview.newrelic.com/sitemap.xml',
```

Ooooo good call on this since the default is to pull from `siteMetadata.siteUrl`. Only thing I'd suggest here is to remove the `www` since we don't serve the site there (we have a redirect in place for it though).